lib/build/bundle.js

```diff
@@ -127,6 +127,33 @@ exports.Bundle = class {
         let currentFile = files[i];
         let sourceMap = buildOptions.sourcemaps ? currentFile.sourceMap : undefined;
 
+        function acquireSourceMapForDependency(file) {
+          if (!file || !file.path) {
+            return;
+          }
+
+          // todo: Make obtaining the directory more robust.
+          let fileDirectory = file.path.substring(0, file.path.lastIndexOf("/") + 1);
+          let converter = Convert.fromMapFileSource(file.contents.toString(), fileDirectory);
+          sourceMap = converter
+            ? converter.sourcemap
+            : null;
+
+          if (sourceMap != null) {
+            let cwd = process.cwd();
+            let fileDirectoryRelativePath = fileDirectory.substring(cwd.length)
+            sourceMap.sourceRoot = fileDirectoryRelativePath;
```

Comment:
Same goes here. `fileDirectoryRelativePath` isn't used anywhere else in this block, so maybe just do `sourceMap.sourceRoot = fileDirectory.substring(process.cwd().length)`?